github/github.go

```diff
@@ -81,6 +81,9 @@ const (
 
 	// https://developer.github.com/changes/2016-06-14-repository-invitations/
 	mediaTypeRepositoryInvitationsPreview = "application/vnd.github.swamp-thing-preview+json"
+
+	// https://developer.github.com/changes/2016-04-21-oauth-authorizations-grants-api-preview/
+	mediaTypeOAuthGrantAuthorizationsPreview = "application/vnd.github.damage-preview"
```

Comment:
The media type string should have a `+json` suffix at the end, like (most) others, see above.

See https://github.com/google/go-github/pull/337#discussion_r66712727 for more details.